lms/views/content_item_selection.py

```diff
@@ -51,6 +51,8 @@ def content_item_selection(context, request):
     lti_h_service.upsert_h_user()
     lti_h_service.upsert_course_group()
 
+    context.js_config.enable_content_item_selection_mode()
```

Comment:
This view previously relied JavaScript defaulting the mode to `"content-item-selection"`. It didn't set any mode in the JS config. Now JSConfig always contains a mode and it defaults to `"basic-lti-launch"`, so this view now has to change it to `"content-item-selection"`